### PR TITLE
welcome-message

### PR DESCRIPTION
**Pull request by Keyren34**
please :(

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,6 +11,12 @@ client.on("ready", () => {
   console.log(`Logged in as ${client.user.tag}!`)
 })
 
+client.on("guildMemberAdd",(member) => {
+  member.send(
+      '**Welcome to our server!:video_game: Please read the rules before joining any text or voice channels.**'
+  )
```
Comment (DavidvanZyl) [inline index.js:17]: New functions should be in new files. Please do some reading around SOLID principles.

Comment (Keyren34) [reply index.js:17]: thanks daddy

Comment (Keyren34) [reply index.js:17]: I'll do better next time

Comment (JoshBHodges) [inline index.js:17]: Well done, you are now learning javascript and git

**Review by JoshBHodges — APPROVED** [review]
thumbs up

**Pull request merged**